app/models/content_type.rb

```diff
@@ -0,0 +1,17 @@
+class ContentType
```

Comment:
Why not make this a module and extend self rather than having a class?  Will we instantiate any ContentType objects?